src/EditorFeatures/CSharpTest/CommentSelection/CSharpToggleLineCommentCommandHandlerTests.cs

```diff
@@ -396,7 +396,7 @@ class C
 {
     void M()
     {
-[|        ////var i = 1;
+        [|////var i = 1;
```

Comment:
This has changed the selection by including the comment markers inside the selection. This is not the behaviour intended.